service.c

```diff
@@ -143,14 +145,13 @@ int main(int argc, char **argv)
 	{
 		AddToMessageLog(TEXT("StartServiceCtrlDispatcher failed."));
 	}
-
+    system("pause");

```

Comment:
Usually, the debugger is responsible for leaving the terminal window open. I wouldn't add this.